ReportServer.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+console.log('Starting Service Fabrik Report App...');
+const lib = require('./lib');
+const routes = lib.routes;
+const errors = require('./lib/errors');
+const logger = lib.logger;
+const HttpServer = require('./HttpServer');
+const FabrikApp = require('./FabrikApp');
+
+const report = FabrikApp.createApp('report', app => {
+  app.get('/', (req, res) => {
+    res.render('index', {
+      title: app.get('title')
+    });
+  });
+  app.use('/admin/report', routes.report);
+});
+
+HttpServer.startServer(report);
+process.on('SIGTERM', HttpServer.notifyShutDown);
```

Comment:
can these event handlers move over into `HttpServer.handleShutdown` & invoked from within `start` method. This is repetitive code which can be part of `HttpServer`